File: output/deepseek_r1/avatar/direct_translation/python/java/codeforces_519_B.java

import java.util.Scanner;
import java.util.Arrays;

public class codeforces_519_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.nextLine();
        int x = Arrays.stream(scanner.nextLine().split(" ")).mapToInt(Integer::parseInt).sum();
        int y = Arrays.stream(scanner.nextLine().split(" ")).mapToInt(Integer::parseInt).sum();
        int z = Arrays.stream(scanner.nextLine().split(" ")).mapToInt(Integer::parseInt).sum();
        System.out.println(Math.abs(x - y));
        System.out.println(Math.abs(y - z));
    }
}

File: output/llama_4_maverick/avatar/algo_based_translation/java/python/successful_translation/codeforces_519_B.py

def main():
    n = int(input())
    a = sum(map(int, input().split()))
    b = sum(map(int, input().split()))
    c = sum(map(int, input().split()))
    x = a - b
    y = b - c
    print(x)
    print(y)

if __name__ == "__main__":
    main()

